cmd2/__init__.py

```diff
@@ -25,6 +25,7 @@
 from .cmd2 import Cmd
 from .constants import COMMAND_NAME, DEFAULT_SHORTCUTS
 from .decorators import with_argument_list, with_argparser, with_argparser_and_unknown_args, with_category
+from .exceptions import Cmd2ArgparseError, SkipPostcommandHooks
```

Comment:
These changes definitely need some sort of CHANGELOG entry

Reply:
- Suggest updating https://cmd2.readthedocs.io/en/latest/features/commands.html#exception-handling to describe the `BaseException` exclusion for error handling, and the behavior when raising `Cmd2ArgparseError` or `SkipPostcommandHooks` exceptions.
- Also suggest updating https://cmd2.readthedocs.io/en/latest/features/hooks.html to describe this new exception handling behavior
- At a minimum, this change should bump our version to 1.1, since we have added new exceptions to the public API. However, I could make a strong case that change to the `do_command` API (changed exception handling behavior) merits a bump to 2.0. We should discuss.

Reply:
Updating those docs and bumping to version to 1.1 sounds like a good idea